src/Numerics/FindRoots.cs

```diff
@@ -86,17 +86,40 @@ public static double OfFunctionDerivative(Func<double, double> f, Func<double, d
         /// </summary>
         public static (Complex, Complex) Quadratic(double c, double b, double a)
         {
-            if (b == 0d)
+            Complex x1, x2;
+            if (a == 0.0)
             {
-                var t = new Complex(-c/a, 0d).SquareRoot();
-                return (t, -t);
+                if (b == 0.0)
+                {
+                    x1 = Complex.Zero / Complex.Zero;  // Complex.NaN;
```

Comment:
Why not write here directly `Complex.NaN`?

Reply:
Because "Complex.NaN" is not available in net48 and netstandard2.0, it is introduced since net5.0, "Complex.Zero / Complex.Zero" or "Complex(double.NaN, double.NaN)" is used as a substitute for "Complex.NaN" in early versions.

Reply:
Ok, understood. Then I would preferably write `Complex(double.NaN, double.NaN)`